xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java

```diff
@@ -57,6 +54,9 @@ protected void handleMessage(JsonNode message) {
       KrakenEventType krakenEvent;
       if (event != null && (krakenEvent = KrakenEventType.getEvent(event.textValue())) != null) {
         switch (krakenEvent) {
+          case pingStatus:
+            LOG.warn("PingStatus received: {}", message);
```

Comment:
Warn is quite high. Does this warrant waking someone up for?

Reply:
changed to info